src/metadata.rs

```diff
@@ -0,0 +1,92 @@
+// Chemfiles, a modern library for chemistry file reading and writing
+// Copyright (C) 2015-2020 Guillaume Fraux -- BSD licensed
+
+use chemfiles_sys::{chfl_format_metadata, chfl_formats_list};
+use errors::check_success;
+use strings;
+
+/// `FormatMetadata` contains metdata associated with one format.
+#[allow(clippy::struct_excessive_bools)]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FormatMetadata {
+    /// Name of the format.
+    pub name: String,
```

Comment:
The C api is built in such a way that the strings in FormatMetadata are actually guaranteed to be `&'static str` in the Rust sense. While not really necessary for any performance reason, it could be nice to expose them as such =)

Reply:
Agree, that's nicer than owned strings.